.promu.yml

```diff
@@ -10,6 +10,8 @@ build:
           path: ./cmd/prometheus
         - name: promtool
           path: ./cmd/promtool
+        - name: tsdb
```

Comment:
Does this change make sense? @SuperQ

Reply:
The change is fine, but I think it's already in master, so I'm not sure why there is a diff here.

Reply:
I was wondering as well, but it doesn't break anything so didn't spend much time to find out the cause.